### ISSUE TITLE
Background changing issues.

### ISSUE THREAD

**Issue by TheOddball** [sapic/sapic#34]
Please comment repros here.

**Comment by alberui** [comment]
IM Hvaing this problem 
![Screenshot_1434343](https://user-images.githubusercontent.com/31510891/68062955-7caeb900-fd05-11e9-95e1-eaa3eadf0beb.png)

**Comment by TheOddball** [comment]
@alberui what did you put in the URL box? What browser and version of it are you using?

**Comment by alberui** [comment]
im using the latest version of google chrome 
![55](https://user-images.githubusercontent.com/31510891/68063187-eda2a080-fd06-11e9-9a83-e1d96bad67d7.png)

**Comment by TheOddball** [comment]
@alberui try hitting random bg and trying again

**Comment by alberui** [comment]
not working either

**Comment by TheOddball** [comment]
@alberui Can you take a screenshot of the javascript console? `ctrl + shift + j`

**Comment by alberui** [comment]
![2](https://user-images.githubusercontent.com/31510891/68064567-16ca2d80-fd15-11e9-9bc7-3189261cd9b0.png)

**Comment by alberui** [comment]
Still waiting for the answer.

**Comment by TheOddball** [comment]
Waiting on @TrueCarry

**Comment by TrueCarry** [comment]
Clear local storage. Won't fix

**Comment by alberui** [comment]
any solution?
